src/coreclr/vm/codeman.cpp

```diff
@@ -6029,6 +6074,17 @@ BOOL ReadyToRunJitManager::GetBoundariesAndVars(
     return TRUE;
 }
 
+BOOL ReadyToRunJitManager::GetRichDebugInfo(
+    const DebugInfoRequest& request,
+    IN FP_IDS_NEW fpNew, IN void* pNewData,
+    OUT ICorDebugInfo::InlineTreeNode** ppInlineTree,
+    OUT ULONG32* pNumInlineTree,
+    OUT ICorDebugInfo::RichOffsetMapping** ppRichMappings,
+    OUT ULONG32* pNumRichMappings)
+{
+    return FALSE;
```

Comment:
Seems like eventually we will want to support rich debug and prejitting. 

Maybe another TODO?

Reply:
Yes, I will work on this in .NET 8 as part of working out a better storage format for the data.